### PR TITLE
W-17281627-mule-version-duke

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/log-tokenization.adoc [hunk 1]
```diff
@@ -9,18 +9,16 @@ Log messages are often difficult to parse in order to apply meaningful tokenizat
...
-Log tokenization is supported on:
+Log tokenization is available for Hybrid and CloudHub deployments running on xref:release-notes::mule-runtime/mule-esb.adoc[supported Mule versions].
 
-* Hybrid apps running on Mule 3.x or 4.x
-* CloudHub apps running on Mule 3.x or 4.x  
-
-== Prerequisites
+== Before You Begin
 
+//TODO: version 1.13 RTF?? Any other versions? Those pages don't exist in latest RTF docs, btw.
```
Comment (dukesphere) [inline modules/ROOT/pages/log-tokenization.adoc:16]: TODO later via different PR: RTF VERSION

File: modules/ROOT/pages/log-points.adoc [hunk 2]
```diff
@@ -9,12 +9,11 @@ The Log Points feature in Anypoint Monitoring allows you to generate logs for ap
...
 
-=== Prerequisites
+=== Before You Begin
 
-To use this feature, you must have the following:
+These prerequisites are required:
 
-//TODO: W-17281627: 3.8.5??? Check Mule version req.
-* Mule 3.8.5 or later
+* A xref:release-notes::mule-runtime/mule-esb.adoc[supported Mule version].
```
Comment (dukesphere) [inline modules/ROOT/pages/log-points.adoc:16]: points here: https://docs.mulesoft.com/release-notes/mule-runtime/mule-esb

File: modules/ROOT/pages/log-tokenization.adoc [hunk 3]
```diff
@@ -9,18 +9,16 @@ Log messages are often difficult to parse in order to apply meaningful tokenizat
 
 == Supported Mule Versions
 
-Log tokenization is supported on:
+Log tokenization is available for Hybrid and CloudHub deployments running on xref:release-notes::mule-runtime/mule-esb.adoc[supported Mule versions].
```
Comment (dukesphere) [inline modules/ROOT/pages/log-tokenization.adoc:12]: points here: https://docs.mulesoft.com/release-notes/mule-runtime/mule-esb

Comment (dukesphere) [reply modules/ROOT/pages/log-tokenization.adoc:12]: PM is good with that.

File: modules/ROOT/pages/monitor-connectors.adoc [hunk 4]
```diff
@@ -9,11 +9,12 @@ You also need the Unified Agent.
 
 == Supported Mule Versions
 
-To monitor connectors, you must use a supported version of Mule:
+To monitor connectors, you must use a xref:release-notes::mule-runtime/mule-esb.adoc[supported Mule version] for your apps:
```
Comment (dukesphere) [inline modules/ROOT/pages/monitor-connectors.adoc:12]: points here: https://docs.mulesoft.com/release-notes/mule-runtime/mule-esb
PM good with this

File: modules/ROOT/pages/monitor-connectors.adoc [hunk 5]
```diff
@@ -9,11 +9,12 @@ You also need the Unified Agent.
...
-To monitor connectors, you must use a supported version of Mule:
+To monitor connectors, you must use a xref:release-notes::mule-runtime/mule-esb.adoc[supported Mule version] for your apps: 
 
-* Hybrid apps running on Mule 4.2.0, 4.1.x, 3.9.x, or 3.8.7
-* CloudHub apps running on Mule 4.1.x or later
-* CloudHub 2.0 apps running on Mule 4.3 or later
+* Hybrid deployments
+* CloudHub and CloudHub 2.0 deployments
+//TODO: RTF DEPLOYMENTS
+//* Runtime Fabric deployments
```
Comment (dukesphere) [inline modules/ROOT/pages/monitor-connectors.adoc:17]: TODO later via different PR: RTF VERSION

Comment (dukesphere) [reply modules/ROOT/pages/monitor-connectors.adoc:17]: [W-17425078](https://gus.lightning.force.com/a07EE000026hrW1YAI)

File: modules/ROOT/pages/log-tokenization.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/log-tokenization.adoc:16]: [W-17425078](https://gus.lightning.force.com/a07EE000026hrW1YAI)

**Review by sathya0 — APPROVED** [review]

**Pull request merged**